README.md

```diff
@@ -232,13 +248,14 @@ Please make sure to include `#gene_id` in the first line. The default exclusion
 
 There are 7 different output files produced by `staramr`:
 
-1. `summary.tsv`:  A summary of all detected AMR genes/mutations/plasmids in each genome, one genome per line.
-2. `detailed_summary.tsv`: A detailed summary of all detected AMR genes/mutations/plasmids in each genome, one gene per line.
+1. `summary.tsv`:  A summary of all detected AMR genes/mutations/plasmids/sequence type in each genome, one genome per line.
+2. `detailed_summary.tsv`: A detailed summary of all detected AMR genes/mutations/plasmids/sequence type in each genome, one gene per line.
 3. `resfinder.tsv`: A tabular file of each AMR gene and additional BLAST information from the **ResFinder** database, one gene per line.
 4. `pointfinder.tsv`: A tabular file of each AMR point mutation and additional BLAST information from the **PointFinder** database, one gene per line.
 5. `plasmidfinder.tsv`: A tabular file of each AMR plasmid type and additional BLAST information from the **PlasmidFinder** database, one plasmid type per line.
-6. `settings.txt`: The command-line, database versions, and other settings used to run `staramr`.
-7. `results.xlsx`: An Excel spreadsheet containing the previous 6 files as separate worksheets.
+6. `mlst.tsv`: A tabular file of each AMR sequence type and it's corresponding locus/alleles, one genome per line.
```

Comment:
Could you change **each AMR sequence type** to **each multi-locus sequence type (MLST)**.

Reply:
Will be fixed in my next commit.